modules/openshift-developer-cli-deploying-the-front-end-component.adoc

```diff
@@ -3,12 +3,27 @@
 // * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
 // * cli_reference/openshift_developer_cli/creating-an-application-with-a-database.adoc
 
+ifeval::["{context}" == "creating-a-multicomponent-application-with-odo"]
+:multi:
+endif::[]
+ifeval::["{context}" == "creating-an-application-with-a-database"]
+:database:
+endif::[]
+
 [id="deploying-the-front-end-component_{context}"]
 
+To create and deploy a front-end component, download the Node.js application and push the source code to your cluster with `{odo-title}`.
```

Comment:
@boczkowska, please put this after the heading.

Reply:
Oops, sorry about that. Fixed.